libsrc/Makefile

```diff
@@ -37,13 +38,14 @@ DRVTYPES = emd \
            ser \
            tgi
 
-OUTPUTDIRS := lib         \
-              $(DRVTYPES) \
-              targetutil  \
-              asminc      \
-              cfg         \
-              include     \
-              $(subst ../,,$(filter-out $(wildcard ../include/*.*),$(wildcard ../include/*)))
+OUTPUTDIRS := lib                 \
+              asminc              \
+              cfg                 \
+              include             \
+              samples             \
```

Comment:
I don't see why the libsrc Makefile should even know that there's something like 'samples'.